Source/WebCore/dom/Document.cpp

```diff
@@ -10057,6 +10057,9 @@ void Document::addTopLayerElement(Element& element)
         if (dialogElement && dialogElement->isModal())
             return;
         auto result = m_autoPopoverList.add(*candidatePopover);
+#if PLATFORM(IOS_FAMILY) && ENABLE(TOUCH_EVENTS)
```

Comment:
Fwiw I can reproduce the issue with WPE Android, so it's possible that just doing this for iOS is too restrictive?

I'm unsure whether the function you call exists on other platforms though.

Reply:
Android might need a separate fix, I'm changing code specific to iOS (that's not open source fwiw)